src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/IntegrationTestBase.kt

```diff
@@ -201,6 +201,9 @@ abstract class IntegrationTestBase {
     conversationRepository.deleteAll()
     reviewRepository.deleteAll()
     reviewScheduleRepository.deleteAll()
+    reviewScheduleHistoryRepository.deleteAll()
```

Comment:
We should probably empty all the tables etc before each test.
We definitely need to do this for `reviewSheduleHistoryRepository` because some of my tests in this PR assert the table is empty